pkg/dependency/dotnet_runtime.go

```diff
@@ -62,6 +64,19 @@ func (d dotnetRuntimeType) getReleaseVersions(release DotnetChannelRelease) []st
 	return []string{release.Runtime.Version}
 }
 
+func (d dotnetRuntimeType) getCPE(version string) (string, error) {
+	parsedVersion, err := semver.NewVersion(version)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse semverL %w", err)
+	}
+
+	productName := ".net"
+	if parsedVersion.LessThan(semver.MustParse("5.0.0-0")) { // use 5.0.0-0 to ensure 5.0.0 previews/RCs use the new `.net` product name
```

Comment:
Can we not use `5.0.0` now that we ignore preview versions? Or does that check not happen until later?

Reply:
`5.0.0` would work but the code filtering out preview versions happens elsewhere. It felt more right for this method to return the correct CPE in every case even if we don't currently call it with preview versions.

Reply:
Makes sense, just wanted to confirm it was on purpose.